packages/contracts/contracts/test/ENSMigration.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+// This is a migration file as suggested here https://docs.ens.domains/deploying-ens-on-a-private-chain#migration-file-example
+// to compile the contracts and make their artifacts available in our tests.
+
+import "@ensdomains/ens-contracts/contracts/registry/ENSRegistry.sol";
+import "@ensdomains/ens-contracts/contracts/resolvers/PublicResolver.sol";
+
+contract Dummy {}
```

Comment:
[As suggested here, we use a migration file](https://docs.ens.domains/deploying-ens-on-a-private-chain#migration-file-example) to make the ENS contracts available in the testing folders.

Unfortunately, the ENS contracts produce a lot of compiler warnings ([see `ENS-compiler-warnings.txt` attached](https://github.com/aragon/core/files/8878182/ENS-compiler-warnings.txt)).
Does anyone know a way how to silence compiler warnings for specific files or has an idea how to handle this?

Reply:
Any suggestions @brickpop @novaknole? If this is solved, this PR Draft can be reviewed and merged.

Reply:
@Michael-A-Heuer one option could be to copy/move the relevant OZ contract files to a `vendor` folder within the repo's codebase, to prevent the declaration shadowing in the first place. The only alternative would be to play with `grep` and filter some of these warnings out, but that's a convenience workaround more than a solution.

The compiler warnings are there for a reason, if they don't affect the end result/code being shipped, we should be good to jump into the review

Reply:
@brickpop should we resolve this conversation and leave the warnings for now, or how do we want to proceed?

Reply:
I suggest so 👍